feg/radius/src/run.sh

```diff
@@ -51,7 +51,7 @@ function pretty {
 }
 
 function test {
-    gotestsum ./...
+    gotestsum -- -p 1 ./...
```

Comment:
Maybe a comment would be helpful here? Something like "Running tests in parallel to avoid radius server cross-talk between tests"

Reply:
The comment would be about running sequentially, not in parallel. I will add something to that effect before merging.

Reply:
Oops, that's what I meant, sorry. Looks good, thanks!